Jenkinsfile

```diff
@@ -0,0 +1,63 @@
+library changelog: false, identifier: 'lib@master', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
+]) _
+
+pipeline {
+    agent {
+        label 'large-amazon'
+    }
+    parameters {
+        choice(
+            choices: [ 'test', 'e2e' ],
+            description: 'Run UI tests?',
+            name: 'RUN_TEST')
+        string(
+            defaultValue: '80.0',
+            description: 'Google Chrome version',
+            name: 'CHROME_VERSION')
+    }
```

Comment:
I am not really sure if parameters will be useful if we make a JenkinsFile, if it was a standalone groovy script which could be used by a Job, then definitely it was a good option.